index.js

```diff
@@ -8,7 +8,7 @@ const statusCodes = {
 };
 
 module.exports = async url => {
-	url = url.replace(/^(?:https?:)?\/\//, '');
+	url = url.substring(url.indexOf('www'), url.indexOf('.com') + 4);
```

Comment:
Would be better to use `new URL(url)` and then get the host.

Reply:
Yes, it would be better. Adding this change now